presentation/src/main/java/com/anytypeio/anytype/presentation/sets/ObjectSetExtension.kt

```diff
@@ -569,4 +536,57 @@ suspend fun DVViewer.getProperTemplateId(
     } else {
         null
     }
-}
\ No newline at end of file
+}
+
+suspend fun ObjectState.DataView.getActiveViewTypeAndTemplate(
+    ctx: Id,
+    activeView: DVViewer?,
+    storeOfObjectTypes: StoreOfObjectTypes
+): Pair<ObjectWrapper.Type?, Id?> {
```

Comment:
is it possible to replace the return type by the following one?

`Pair<ObjectWrapper.Type, Id>?`

`Pair(null, null)` looks a little bit weird.

Reply:
Sorry, no, I can't find any good solutions here.